docs/polkadot/implementors-guide/guide.md

```diff
@@ -0,0 +1,544 @@
+## The Polkadot Parachain Host Implementers' Guide
+
+## Ramble / Preamble
+
+This document aims to describe the purpose, functionality, and implementation of a host for Polkadot's _parachains_. It is not for the implementor of a specific parachain but rather for the implementor of the Parachain Host, which provides security and advancement for constituent parachains. In practice, this is for the implementors of the Relay Chain. 
+
+There are a number of other documents describing the research in more detail. All referenced documents will be linked here and should be read alongside this document for the best understanding of the full picture. However, this is the only document which aims to describe key aspects of Polkadot's particular instantiation of much of that research down to low-level technical details and software architecture.
+
+## Table of Contents
+* [Origins](#Origins)
+* [Parachains: Basic Functionality](#Parachains-Basic-Functionality)
+* [Architecture](#Architecture)
+  * [Node-side](#Architecture-Node-side)
+  * [Runtime](#Architecture-Runtime)
+* [Processes](#Processes)
+  * [Overseer](#Overseer-Process)
+  * [Candidate Backing](#Candidate-Backing)
+* [Data Structures and Types](#Data-Structures-and-Types)
+* [Glossary / Jargon](#Glossary)
+
+
+## Origins
+
+Parachains are the solution to a problem. As with any solution, it cannot be understood without first understanding the problem. So let's start by going over the issues faced by blockchain technology that let to us beginning to explore the design space for something like parachains.
+
+#### Issue 1: Scalability
+
+It became clear a few years ago that the transaction throughput of simple Proof-of-Work (PoW) blockchains such as Bitcoin, Ethereum, and myriad others was simply too low. [TODO: PoS, sharding, what if there were more blockchains, etc. etc.]
+
```

Comment:
A bit on security also:

```suggestion

The other concern is the vulnerability of newly created PoW blockchains to  51% attacks. An amount of existing mining power in the world is finite and it is unlikely that a newly created blockchain would attract as much mining power as the existing popular PoW blockchains. This fact makes 51% attacks against our newly created blockchain relatively cheap. The estimated cost of 1 hour attack on Bitcoin is $700000/hour however for a a huge number of less popular PoW blockchains the estimate cost of an attack under $100/hour (at the time of writing of this document). It is obvious that implementing a new blockchain with a PoW consensus is orthogonal to creating a system with a reasonable level of security.

```

Reply:
Do you have a source on that information? It may not fit in the flow of a "scalability" section, but there could be another section on PoS vs PoW.

Reply:
> Recently there have been a number of 51% attacks including a high profile attack against Bitcoin Gold where $18 Million was stolen.

https://www.crypto51.app/